src/Sulu/Component/Content/Tests/Unit/Document/Subscriber/NavigationContextSubscriberTest.php

```diff
@@ -44,7 +44,7 @@ public function setUp()
         $this->document = $this->prophesize(NavigationContextBehavior::class);
         $this->metadata = $this->prophesize(Metadata::class);
         $this->event = $this->prophesize(MetadataLoadEvent::class);
-        $this->subscriber = new NavigationContextSubscriber($this->encoder->reveal());
```

Comment:
Don't know why this was here before, but it was breaking the test for me on PHP 7.1.